### ISSUE TITLE
Some input XML in tests refer to nonexistent xsd

### ISSUE THREAD

**Issue by PDoakORNL** [QMCPACK/qmcpack#4532]
**Describe the bug**
Some tests attempt to does some XML namespace and schema setup in their input.
This is unnecessary and the schema is nonexistent.  It also implies that QMCPACK inputs could be
not just well-formed XML but valid i.e. check against some schema.  As far as I know that was never true.

This is not a bug per say for QMCPACK but if someone put our XML into a parser or editor that attempted to do validation this would create errors.  It is also confusing to users and developers.

**To Reproduce**
open a performance test input

``` XML
<simulation xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.mcc.uiuc.edu/qmc/schema/molecu.xsd">
<!-- ...-->
</simulation>
```

**Remedy**

```XML
<simulation>
<!-- ...-->
</simulation>
```

**Comment by markdewing** [comment]
The schema files are still in the repository at https://github.com/QMCPACK/qmcpack/tree/develop/schema , but we don't use them.  These could get removed as well.

**Comment by prckent** [comment]
I agree about the removal and deletion of references to this. The schema is out of date by years. There are at least 61 xml files referencing the xsd in the repo (via `grep -R -i -l ".xsd" --include "*.xml"|wc -l`)
